service/s3control/endpoint_test.go

```diff
@@ -80,51 +80,31 @@ func TestEndpoint_OutpostAccessPointARN(t *testing.T) {
 			expectedHeaderForAccountID: true,
 			expectedHeaderForOutpostID: "op-01234567890123456",
 		},
-		"Outpost AccessPoint with client region as Fips": {
+		"Outpost AccessPoint with client region as FIPS": {
 			bucket: "arn:aws-us-gov:s3-outposts:us-gov-east-1:123456789012:outpost:op-01234567890123456:accesspoint:myaccesspoint",
 			config: &aws.Config{
 				EndpointResolver: endpoints.AwsUsGovPartition(),
 				Region:           aws.String("us-gov-east-1-fips"),
 			},
-			expectedErr: "InvalidARNError: resource ARN not supported for FIPS region",
+			expectedErr: "client configured for fips but cross-region resource ARN provided",
```

Comment:
same comment as above. Since using ARNs isn't valid when client/request is configured with fips, should we be explicit in the error thrown?